pkg/sti/script/installer.go

```diff
@@ -108,6 +109,20 @@ func (s *handler) download(scripts []api.Script, workingDir string) (bool, error
 		}
 	}
 
+	// check for the scripts inside the sources
+	scriptsDir := filepath.Join(workingDir, "/upload/src/.sti/bin")
+	if s.fs.Exists(scriptsDir) {
+		for _, script := range scripts {
+			file := filepath.Join(scriptsDir, string(script))
+			if s.fs.Exists(file) {
+				downloads[script] <- true
+			} else {
+				errs[script] <- errors.NewScriptDownloadError(script, nil)
```

Comment:
This is analogical to what we do in `downloadAsync` (see https://github.com/openshift/source-to-image/blob/master/pkg/sti/script/installer.go#L95-L109), where we'll complain about errors if the file cannot be found. The result is then processed in [executor.go](https://github.com/openshift/source-to-image/blob/master/pkg/sti/executor.go#L104-L111) where depending if the scripts are required or not, fails or warns a user.